### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by krista-bennett** [Turing-Complete-Game/Suggestions-and-Issues#247]
### What happened?

(Note: someone has also filed this bug on the Steam forums, but since it's not in the tracker and I hit it too, I figured I'd post here)

    In Storage Cracker no matter what value I send to output I never get 1 from the input. Even if I send 255 to output 
    I still get 0 as the input

I've tried it with 63 and 127 and 255 (I presumed this was too large, but hey - the range is never specified), but it doesn't matter. A brute force solution counting down from any of these as maximum values never gives 1 as input, and I've checked that input and output are enabled on the respective tics. No dice. Input is always 0.

I'm running 0.418 Beta, but I don't know what version that maps to below.

### Version

Latest

### Operating System

Windows 10

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kerams** [comment]
Yep, saw this too. It prevents the fast solution of halving the search range.

**Comment by MegaIng** [comment]
Does this still happen?

**Comment by Pixpeg** [comment]
Yes this is still an issue. Tried sending values 63 and 255 to the OUT but no "1" on input.
0.599 Beta

**Comment by MegaIng** [comment]
So this exact circuit does not produce a 1 on the second tick for you?

![image](https://user-images.githubusercontent.com/26136419/159351820-5c7185d4-4454-4028-ad61-03649fac493a.png)

**Comment by Pixpeg** [comment]
Oooops! My error - I hadn't enabled the Input node - Big sorry :(

**Comment by GufNZ** [comment]
Close?
